includes/options/views/class-amp-setup-wizard-submenu-page.php

```diff
@@ -74,13 +110,22 @@ public function enqueue_assets( $hook_suffix ) {
 			true
 		);
 
+		wp_enqueue_style(
+			self::JS_HANDLE,
+			amp_get_asset_url( "css/amp-setup-compiled{$rtl}.css" ),
+			[],
+			AMP__VERSION
+		);
+
 		wp_add_inline_script(
 			self::JS_HANDLE,
 			sprintf(
 				'var ampSetup = %s;',
 				wp_json_encode(
 					[
-						'APP_ROOT_ID' => self::APP_ROOT_ID,
+						'APP_ROOT_ID'           => self::APP_ROOT_ID,
+						'EXIT_LINK'             => admin_url( 'admin.php?page=' . AMP_Options_Manager::OPTION_NAME ),
+						'OPTIONS_REST_ENDPOINT' => rest_url( 'amp-wp/v1/options' ),
```

Comment:
Depending on how the outcome of the REST API class above, this endpoint will have to be updated.